NeoML/test/src/MobileNetV2BlockTest.cpp

```diff
@@ -174,7 +174,7 @@ static void mobileNetV2BlockTestImpl( unsigned int seed, int freeTermMask, float
 
 	CPtr<CDnnBlob> expectedBlob = expectedSink->GetBlob();
 	CPtr<CDnnBlob> actualBlob = actualSink->GetBlob();
-	CompareBlobs( *expectedBlob, *actualBlob );
```

Comment:
`ASSERT_EXPR` is an NeoML macro which is used for invalid operations (attemt to allocate `0` bytes, unerachable case statement, smth like that).

For the results of the tests we use gtest macro `ASSERT/EXPECT _ EQ/FLOAT_EQ/TRUE/FALSE` etc.

Reply:
There is the same mistake in CtcTest

Reply:
fixed!